From b33f780b8ad41f0ee6c8b7f75cb4676d1b1035cf Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Thu, 9 Aug 2018 01:16:36 -0700
Subject: [PATCH 06/13] [TEMP] lineage-sdk: Comment out unbuildable code

*) Comment out code that is unbuildable owing to missing features
   elsewhere.

Change-Id: Idcbcf0ce902d9edae3383e0e5eacb35380b6465a
---
 sdk/src/java/org/lineageos/internal/util/ActionUtils.java | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sdk/src/java/org/lineageos/internal/util/ActionUtils.java b/sdk/src/java/org/lineageos/internal/util/ActionUtils.java
index c00cc17..a95f2cc 100644
--- a/sdk/src/java/org/lineageos/internal/util/ActionUtils.java
+++ b/sdk/src/java/org/lineageos/internal/util/ActionUtils.java
@@ -121,6 +121,7 @@ public class ActionUtils {
             throws RemoteException {
         final String defaultHomePackage = resolveCurrentLauncherPackage(context, userId);
         final ActivityManager am = (ActivityManager) context.getSystemService(Context.ACTIVITY_SERVICE);
+/*
         final List<ActivityManager.RecentTaskInfo> tasks = am.getRecentTasksForUser(5,
                 ActivityManager.RECENT_IGNORE_UNAVAILABLE, userId);
 
@@ -135,6 +136,7 @@ public class ActionUtils {
                 return tasks.get(i);
             }
         }
+*/
 
         return null;
     }
-- 
2.17.1

